### ISSUE TITLE
Improve Accessibility

### ISSUE THREAD

**Issue by jesse-ashmore** [JeeZeh/React-Irish-Rail#19]
Firefox Dev Tools Accessibility Review has highlighted a number of issues such as missing labels, non-selectable buttons, and contrast issues.

These should be fixed so that the a11y review passes with flying colours. Speaking of which, the website is also not colourblind-friendly with the usage of Red being particularly unclear.
